### PR TITLE
<TimeInput/> replace css with stylable

### PR DESCRIPTION
**Pull request by israelabuhay**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6177

File: src/TimeInput/TimeInput.js [hunk 1]
```diff
@@ -297,8 +295,8 @@ export default class TimeInput extends Component {
         ref="input"
         rtl={rtl}
         value={text}
-        className={classNames({
-          [styles.input]: width === 'auto',
+        className={st(classes.input, {
```
Comment (gendeld) [inline src/TimeInput/TimeInput.js:298]: This sets `input` element (not the container) width to 55px even when `width="100%`.
it should be
```
className={width === 'auto' ? classes.input : undefined}
```

File: src/TimeInput/TimeInput.st.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+.time:focus {
+  outline: value(timeFocusOutlineColor);
+}
+
+.time:hover {
+   outline: value(timeHoverOutlineColor);
+ }
+
+.time:rtl {
```
Comment (gendeld) [inline src/TimeInput/TimeInput.st.css:63]: You shouldn't have duplicate states on parent and child,
you can do:
```
.root:rtl .time {
```
and get rid of the duplicate state in the st file and the component file

File: src/TimeInput/TimeInput.st.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+:import {
+  -st-from: '../Input/Input.st.css';
+  -st-default: Input;
+}
+
+:vars {
+  hoverAmPmIndicationColor: value(B10);
+  timeHoverOutlineColor: 1px solid blue ;
```
Comment (gendeld) [inline src/TimeInput/TimeInput.st.css:13]:
```suggestion
  timeHoverOutlineColor: 1px solid blue;
```

File: src/TimeInput/TimeInput.st.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  timeFocusOutlineColor: 1px solid black ;
```
Comment (gendeld) [inline src/TimeInput/TimeInput.st.css:14]:
```suggestion
  timeFocusOutlineColor: 1px solid black;
```

File: src/TimeInput/TimeInput.st.css [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+.root:disabled {
+  cursor: default;
+  pointer-events: none;
+}
+
+.root:rtl {
+  flex-direction: row-reverse;
+}
+.input {
```
Comment (gendeld) [inline src/TimeInput/TimeInput.st.css:33]:
```suggestion

.input {
```

File: src/TimeInput/TimeInput.js [hunk 6]
```diff
@@ -316,23 +312,26 @@ export default class TimeInput extends Component {
...
-            hover: hover && !focus,
-            rtl,
-            [styles.stretch]: width === '100%',
-          })}
+          className={st(
+            classes.time,
+            {
+              focus,
+              hover: hover && !focus,
+              rtl,
```
Comment (gendeld) [inline src/TimeInput/TimeInput.js:330]: Remove duplicate state

File: src/TimeInput/TimeInput.st.css [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  -st-extends: Input;
+}
+
+.input::input {
+  width: 55px;
+}
+
+.time {
+  -st-states: focus, hover, rtl, stretch;
```
Comment (gendeld) [inline src/TimeInput/TimeInput.st.css:43]:
```suggestion
  -st-states: focus, hover, stretch;
```

**Review by gendeld — APPROVED** [review]

**Pull request merged**